eth2near/utilities/src/eth_rpc_client.rs

```diff
@@ -0,0 +1,88 @@
+use crate::{
+    primitives::U256,
+    types::{BlockHeader, TransactionReceipt},
+};
+use reqwest::blocking::Client;
+use serde::Deserialize;
+use serde_json::{json, Value};
+use std::error::Error;
+
+pub struct EthRPCClient {
+    endpoint_url: String,
+    client: Client,
+}
+
+impl EthRPCClient {
```

Comment:
The reason we decided against using that one is because it depends on types from `contracts/` folder and we wanted to avoid that dependency. Perhaps the new client can replace `eth1_rpc_client` in the future